IDEAS/Examples/IBPSA/SingleZoneResidentialHydronic.mo

```diff
@@ -1,12 +1,17 @@
 within IDEAS.Examples.IBPSA;
 model SingleZoneResidentialHydronic
   "Single zone residential hydronic example model"
+  import IBPSA;
```

Comment:
We should not need this import as we should only use IDEAS models.

Reply:
Ups... https://github.com/open-ideas/IDEAS/pull/1143/commits/8c7e1b16ccbc9b4e5389dfce926faa016619bb54